front/src/config/demo.js

```diff
@@ -263,7 +268,7 @@ const data = {
       }
     ]
   },
-  'get /api/v1/room/living-room?expand=temperature': {
+  'get /api/v1/room/living-room?expand=temperature,devices': {
```

Comment:
Simple question, why did you add "devices" here?

Reply:
Browser console complain about this query